lib/fs-driver.js

```diff
@@ -85,25 +85,20 @@ class FsDriver {
    * @param  {Function} cb Callback to execute when the teardown is complete.
    */
   teardown (commit, cb) {
-    const rootPath = this.escapePath(this.root)
-    const backupPath = this.escapePath(this.root + '.bak')
-    const workingPath = this.escapePath(this.workingPath)
-    const resultsPath = this.escapePath(this.resultsPath)
-
-    let commands = [
-      `rm -rf ${workingPath}`,
-      `rm -rf ${resultsPath}`
-    ]
+    const rootPath = this.root
+    const backupPath = this.root + '.bak'
+    const workingPath = this.workingPath
+    const resultsPath = this.resultsPath
 
     if (commit) {
-      commands = [
-        `mv ${rootPath} ${backupPath}`,
-        `mv ${workingPath} ${rootPath}`,
-        `rm -rf ${backupPath} ${resultsPath}`
-      ]
+      async.series([
```

Comment:
Also `exec` just takes a straight string. `execFile` takes a command name and then an array of arguments. So I couldn't do the `&&` trick to just do a bunch of commands in sequence, I chose to use `async` because it was cleaner for this particular thing then chaining a bunch of promises.